resources/js/components/catalog-new/Filter.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+    <div class="tw-relative">
+        <slot />
+    </div>
+</template>
+
+<script>
+import NewCustomSelect from './NewCustomSelect.vue'
+import NewMobileCustomSelect from './NewMobileCustomSelect.vue'
+
+export default {
+    props: {
+        authors: Array,
+    },
+    components: { NewCustomSelect, NewMobileCustomSelect },
+    data() {
+        return {
+            openedFilter: null,
+            isExtendedOpen: true,
+        }
+    },
+    computed: {
+        filters() {
+            return {
+                authors: {
+                    list: this.authors.map((author) => ({
+                        ...author,
+                        checked: this.isSelectedMultiSelect('authors', author.name),
+                    })),
+                },
+                someOtherFilter: { list: [] },
+            }
+        },
+    },
+    methods: {
+        isSelectedMultiSelect(filterName, name) {
+            const urlQuery = this.$route.query
+            return urlQuery[filterName] && urlQuery[filterName].includes(name)
+        },
+        toggleIsExtendedOpen() {
+            this.isExtendedOpen = !this.isExtendedOpen
+        },
+        clearSelection(filterName) {
+            const { [filterName]: removedFilterName, ...queryWithoutFilterName } = this.$route.query  
+            this.$router.push({
+                path: 'katalog-new',
+                query: queryWithoutFilterName,
+            })
```

Comment:
I think this should work without the `path` parameter
```suggestion
            this.$router.push({
                query: queryWithoutFilterName,
            })
```